projects/portal/src/app/models/keys/key-select.guard.spec.ts

```diff
@@ -0,0 +1,153 @@
+import { KeySelectDialogService } from './key-select-dialog.service';
+import { KeySelectGuard } from './key-select.guard';
+import { KeyType } from './key.model';
+import { KeyService } from './key.service';
+import { KeyStoreService } from './key.store.service';
+import { TestBed } from '@angular/core/testing';
+import { Router } from '@angular/router';
+import { BehaviorSubject } from 'rxjs';
+
+window.alert = jest.fn();
+
```

Comment:
This may affect other tests. Please refer here.

https://stackoverflow.com/questions/53611098/how-can-i-mock-the-window-alert-method-in-jest

Reply:
Sorry, I saw this article but could not notice this contamination. fixed.